source/MaterialXGenShader/ColorManagementSystem.h

```diff
@@ -78,17 +69,11 @@ class ColorManagementSystem
 
   protected:
     /// Protected constructor
-    ColorManagementSystem(const string& configFile);
+    ColorManagementSystem();
       
     /// Returns an implementation name for a given transform
     virtual string getImplementationName(const ColorSpaceTransform& transform) const = 0;
 
-    /// Register a node implementation for a given color space transformation.
-    void registerImplementation(const ColorSpaceTransform& transform, CreatorFunction<ShaderNodeImpl> creator);
-
-    Factory<ShaderNodeImpl> _implFactory;
-    vector<string> _registeredImplNames;
-    string _configFile;
```

Comment:
Stripped out methods and members that are currently not used, to avoid publishing unused API. We can reintroduce this later once we get around to implement OCIO support.